test/ClangModules/enum-objc.swift

```diff
@@ -0,0 +1,12 @@
+// FIXME: @objc(Renamed) enums should work
```

Comment:
FWIW, you should be able to add the flag -enable-swift-name-lookup-tables to this test to get it working now. The Swift name lookup tables aren't on by default just yet, but they address this problem already.